deploy/registration-service.yaml

```diff
@@ -48,6 +48,7 @@ objects:
         resources:
           - secrets
           - configmaps
+          - serviceaccounts
```

Comment:
Same question as in the host operator PR. Why do you need it?

Reply:
You're right, it's not needed. Removed it